docs/mockups/05-redesign-hybrid-field-console.html

```
@@ -0,0 +1,463 @@
+          </div>
+        </div>
+      </div>
+
+      <div class="f-foot">
+        <span>© 2026 ramie fathy, md</span>
+        <span class="fl"><a href="#fRoot">email</a><a href="#fRoot">scholar</a><a href="#fRoot">linkedin</a><a href="#fRoot">github</a></span>
+      </div>
+
+      <div class="f-overlay" id="fOverlay">
+        <div class="f-pal" role="dialog" aria-label="Navigation palette">
+          <input id="fPalInput" type="text" placeholder="Type a destination…" aria-label="Filter destinations">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)